files/conf/docker-daemon.json

```diff
@@ -0,0 +1,5 @@
+{
+    "registry-mirrors": [
+        "https://quay.io"
+    ]
```

Comment:
I felt adding a list of mirrors to config file was more readable than adding a bunch of flags to `docker.service`. This is because the order of mirrors is v. important here, and it's not immediately apparent when using flags.

Reply:
Should we also add conditional aliyun?

Reply:
Or later? CC @JosephSalisbury

Reply:
This would cause `dockerd` to resolve in this order (for the record):
1. quay.io
1. aliyun.com
1. docker.io

Reply:
I'd rather worry about aliyun later - let's get it working with just quay / outside china first

Reply:
Ack, reverted commits adding aliyun.